resources/wikia/modules/scriptwriter.js

```diff
@@ -68,10 +68,14 @@ define('wikia.scriptwriter', [
 			win.postscribe(
 				params.element,
 				params.html,
-				{beforeWrite: psBeforeWrite, done: params.callback}
+				{
+					autoFix: false,
```

Comment:
Is there postscribe documentation somewhere? I wonder if we could leave a comment here explaining what's the `autoFix` is.